src/angular/components/send/send.component.ts

```diff
@@ -116,12 +119,14 @@ export class SendComponent implements OnInit {
             clearTimeout(this.searchTimeout);
         }
         if (timeout == null) {
+            this.hasSearched = this.searchService.isSearchable(this.searchText);
```

Comment:
Perhaps I misunderstood the purpose of `this.hasSearched`. My understanding is this was going to track whether or not a search was performed, where here it appears to be tracking whether or not a search _can_ be performed.

Perhaps this needs to go below `this.applyTextSearch();`? Not entirely sure.

Reply:
From what I can see - you're right on both accounts :D 

The variable name `hasSearched` is signifying that the function `search(...)` has been called but its context does indeed prove that the search text _can_ and _should_ be acted upon via the `searchService` (next line).

I'm okay with leaving it or changing the name to be more concise!

Reply:
I dunno, `canAndLikelyHasSearched` seems a bit too wordy. This is probably okay as-is now that I understand the context a bit more. As long as nobody modifies other code later in the UI to cascade UI changes based on this bit where the search function may fail, etc. and/or have unexpected results (timing/sequence). But probably good for now; just some code comments would be good I think where it's declared (in this instance).

Reply:
I <3 comments